### PR TITLE
Add note about tracking issues we work on to README

### PR DESCRIPTION
**Pull request by stepanstipl**
Adds a note about tracking issues we work on as per our standup discussion and action item

### CODE REVIEW

File: gcp/README.md [hunk 1]
```diff
@@ -289,6 +289,8 @@ There may be a situation, when we want to roll back entire DB data set to anothe
 
 * Avoid deploying changes near the end of your work day.
 * Avoid deploying on Friday or prior to a holiday.
+* Create JIRA tickets for issues (mainly CI/CD) we encounter and fix - to track the
+  work done and to have data for future reference
```
Comment (natarajaya) [inline gcp/README.md]: The dot is missing at the end of the sentence.

**Comment by natarajaya** [comment]
LGTM

**Comment by mrtyler** [comment]
LGTM

File: gcp/README.md (hunk 1 above)
Comment (stepanstipl) [reply gcp/README.md]: 👍 thanks, dot added

**Pull request merged**